README.md

```diff
@@ -27,25 +27,24 @@ your workflow more fluent.
 ## Building, Testing, and Installation
 
 You'll need the following dependencies:
-* meson
-* libgee-0.8-dev
-* libgranite-dev
-* libgtksourceview-4-dev
-* libwebkit2gtk-4.0-dev
-* libjson-glib-dev
-* libxml2-dev
-* libsqlite3
-* libhandy-1-dev
-* duktape-dev
-* valac
-
-Run `meson build` to configure the build environment. Change to the build directory and run `ninja test` to build
+* flatpak-builder
+
+If you are not on elementary OS then you may also have to install the `flatpak` package before compiling.
+
+To build and install, use `flatpak-builder`.
+
+    flatpak-builder buildir com.github.treagod.spectator.yml --install --user --force-clean
+
+Then you can run the app with `flatpak run com.github.treagod.spectator`.
+
+Once you are done testing and want to remove the app, run `flatpak uninstall com.github.treagod.spectator`.
+
+If you instead want to completely remove the both the app and its user data, run `flatpak uninstall com.github.treagod.spectator --delete-data`.
```

Comment:
One **the** to much: remove both the app and..

Reply:
I removed the extra 'the' since it was the only error (I wanted to say 'If you instead want to completely remove both the app and its user data')